engine/src/player-legacy.cpp

```diff
@@ -218,6 +218,8 @@ MCPlayer::MCPlayer()
 	userCallbackStr = NULL;
 	formattedwidth = formattedheight = 0;
 	loudness = 100;
+	dontuseqt = MCdontuseQT;
+	usingqt = !MCdontuseQT;
```

Comment:
this should be initialized to false by default - it doesn't mean anything until the movie is prepared